.github/ISSUE_TEMPLATE/challenge-program.md

```diff
@@ -1,27 +1,11 @@
 ---
 name: "\U0001F947 Propose a Challenge Program task"
 about: As a developer, I want to propose a Challenge Program task.
-labels: challenge-program-2
 ---
 
-## Description
+You can choose from the following Challenge Program(open the URL):
 
-## Score
+- [challenge-program-high-performance](https://github.com/pingcap/tidb/issues/new?labels=challenge-program%2Chigh-performance&template=challenge-program-high-performance.md)
+- [challenge-program-hptc](https://github.com/pingcap/tidb/issues/new?labels=challenge-program%2Chptc&template=challenge-program-hptc.md)
```

Comment:
what is `hptc` short for?

Reply:
https://github.com/pingcap/community/blob/master/challenge-programs/high-performance-tidb-challenge.md

Reply:
`high-performance` for high performance course.

Reply:
could you prepare a different template for each type of issue? It's better to use full names instead of just abbr to eliminate confusion for new contributors.